_includes/footer.html

```diff
@@ -0,0 +1,24 @@
+<footer>
+  <div>
+    <h1>
+      <a href="mailto:rubyuruguay@ruby.uy">rubyuruguay@ruby.uy</a>
+    </h1>
+    <h2>
+      <a href="tel:+59898455500">+598 984 555 00</a>
```

Comment:
About the 📞 . Should we have something or just remove it?

Reply:
🤣 I wouldn't put my number at least. 🔪

Reply:
haha ok, I'll remove it then.